backend/chainlit/cache.py

```diff
@@ -19,7 +19,7 @@ def init_lc_cache():
         if config.project.lc_cache_path is not None:
             set_llm_cache(SQLiteCache(database_path=config.project.lc_cache_path))
 
-            if not os.path.exists(config.project.lc_cache_path):
+            if os.path.exists(config.project.lc_cache_path):
```

Comment:
I'm not sure to understand why you've removed this?

Reply:
it is because previous line `st_llm_cache` generates sqlite db file at path
```python
set_llm_cache(SQLiteCache(database_path=config.project.lc_cache_path))
```
consequently in next line we want to inform cache was generated by testing if path exists and not the opposite. I hope I was clear - was trying not to use too many "if", "not" in the explaination